### PR TITLE
Update contact information

### PR DESCRIPTION
**Pull request by jkingdon**
There are many places on the web site which contain Norm's email address and this doesn't fix all of them but most notably it fixes email.html (which a lot of things link to) and a number of the references I was able to find (especially ones with a call to action such as "please report problems to").

### CODE REVIEW

**Review by digama0 — APPROVED** [review]

File: email.html [hunk 1]
```diff
@@ -37,71 +31,23 @@
...
-<P>Norman Megill
-<BR>93 Bridge St.
-<BR>Lexington, MA 02421-7946
-<BR> USA
+<ol>
+  <li>Via the <a href="mm-mailing-list.html" >metamath mailing list</a>.</li>
+  <li>By creating a github issue (most often in the
+  <a href="https://github.com/metamath/set.mm" >set.mm</a> repository but we
+  also list related repositories at <a href="other.html" >other.html</a> and
+  some are in the <a href="https://github.com/metamath" >the metmath github
```
Comment (david-a-wheeler) [inline email.html]: Nit: "metmath" should be "Metamath" here. Also, GitHub has capital G and H.

**Comment by david-a-wheeler** [comment]
Thanks so much! I did find a few nits in the proposed changes, would you mind fixing those?

**Comment by david-a-wheeler** [comment]
Looks great! Let's merge!

File: email.html (hunk 1 above)
Comment (jkingdon) [reply email.html]: Oooh, good catch on the missing "a".

As for capitalization, I wasn't sure how globally you meant this so I capitalized Metamath and GitHub everywhere in this file.  I don't have opinions myself (at least, not opinions strong enough to think about it very hard).

**Pull request merged**